### PR TITLE
KS-SK1142-AS-update-NDD-Incoming-TRSF-for-Domain

### PR DESCRIPTION
**Pull request by Kazarus4**
Updates the paragrph after techlead asking

Need peer rv before translation on the same branch

### CODE REVIEW

**Review by OVH-Team-Guides — APPROVED** [review]

**Comment by Kazarus4** [comment]
All translations are done. This except duplication for EN subs (only ENGB is translated).

Need proof for all subs before EN duplication and merge.

File: pages/web/domains/transfer_incoming_generic_domain/guide.es-es.md [hunk 1]
```diff
@@ -1,29 +1,31 @@
...
 
 La transferencia de un dominio permite cambiar el agente registrador que lo gestiona. Puede transferir su dominio a OVHcloud creando un pedido. Esta operación suele tardar entre uno y diez días.
 
-**Esta guía explica cómo transferir un dominio genérico a OVHcloud.**
+**Descruba cómo transferir un dominio genérico a OVHcloud.**
 
 > [!warning]
 >
-> Si el dominio que está siendo modificado se encuentra registrado en OVHcloud, la transferencia entrante de dominio no es el procedimiento adecuado. Este procedimiento solo se aplica al cambio de dominio registrado (OVHcloud).
+> El *agente registrador* de un dominio representa la organización o proveedor acreditado en la que el dominio está registrado/suscrito por un particular, una asociación o una organización. Renovar la suscripción de su dominio (generalmente una vez al año) es a través del mismo agente registrador*.
```
Comment (OVH-Team-Guides) [inline pages/web/domains/transfer_incoming_generic_domain/guide.es-es.md]: broken italic

File: pages/web/domains/transfer_incoming_generic_domain/guide.es-es.md [hunk 2]
```diff
@@ -1,29 +1,31 @@
 ---
 title: 'Transferir un dominio a OVHcloud'
 slug: transferir-un-dominio-generico
-excerpt: 'Cómo realizar la transferencia de un dominio a OVHcloud'
+excerpt: 'Descruba cómo realizar la transferencia de un dominio a OVHcloud'
```
Comment (OVH-Team-Guides) [inline pages/web/domains/transfer_incoming_generic_domain/guide.es-es.md]:
```suggestion
excerpt: 'Descubra cómo realizar la transferencia de un dominio a OVHcloud'
```

File: pages/web/domains/transfer_incoming_generic_domain/guide.es-es.md [hunk 3]
```diff
@@ -1,29 +1,31 @@
...
+**Última actualización: 08/06/2023**
 
 ## Objetivo
 
 Si tiene un dominio registrado con otro proveedor y quiere transferirlo a OVHcloud, puede transferirlo a OVHcloud.
 
 La transferencia de un dominio permite cambiar el agente registrador que lo gestiona. Puede transferir su dominio a OVHcloud creando un pedido. Esta operación suele tardar entre uno y diez días.
 
-**Esta guía explica cómo transferir un dominio genérico a OVHcloud.**
+**Descruba cómo transferir un dominio genérico a OVHcloud.**
```
Comment (OVH-Team-Guides) [inline pages/web/domains/transfer_incoming_generic_domain/guide.es-es.md]:
```suggestion
**Descubra cómo transferir un dominio genérico a OVHcloud.**
```

File: pages/web/domains/transfer_incoming_generic_domain/guide.es-us.md [hunk 4]
```diff
@@ -1,17 +1,17 @@
 ---
 title: 'Transferir un dominio a OVHcloud'
 slug: transferir-un-dominio-generico
-excerpt: 'Cómo realizar la transferencia de un dominio a OVHcloud'
+excerpt: 'Descruba cómo realizar la transferencia de un dominio a OVHcloud'
```
Comment (OVH-Team-Guides) [inline pages/web/domains/transfer_incoming_generic_domain/guide.es-us.md]:
```suggestion
excerpt: 'Descubra cómo realizar la transferencia de un dominio a OVHcloud'
```

File: pages/web/domains/transfer_incoming_generic_domain/guide.es-us.md [hunk 5]
```diff
@@ -23,7 +23,9 @@ La transferencia de un dominio permite cambiar el agente registrador que lo gest
 
 > [!warning]
 >
-> Si el dominio que está siendo modificado se encuentra registrado en OVHcloud, la transferencia entrante de dominio no es el procedimiento adecuado. Este procedimiento solo se aplica al cambio de dominio registrado (OVHcloud).
+> El *agente registrador* de un dominio representa la organización o proveedor acreditado en la que el dominio está registrado/suscrito por un particular, una asociación o una organización. Renovar la suscripción de su dominio (generalmente una vez al año) es a través del mismo agente registrador*.
```
Comment (OVH-Team-Guides) [inline pages/web/domains/transfer_incoming_generic_domain/guide.es-us.md]: broken italic

Comment (OVH-Team-Guides) [inline pages/web/domains/transfer_incoming_generic_domain/guide.es-us.md]: Maybe we should keep the "Registrar" termn in all translations ?

File: pages/web/domains/transfer_incoming_generic_domain/guide.it-it.md [hunk 6]
```diff
@@ -23,7 +23,9 @@ L’operazione di trasferimento consente di spostare un dominio da un Registrar
 
 > [!warning]
 >
-> Se il dominio in corso di modifica è registrato in OVHcloud, il trasferimento in entrata del dominio non è la procedura appropriata. Questa procedura si applica esclusivamente alla modifica del dominio registrato (OVHcloud).
+> L*Registrar* di un dominio rappresenta l'organizzazione/provider accreditata presso la quale il dominio è registrato/sottoscritto da un privato, un'associazione o un'organizzazione. È presso lo stesso *Registrar* che rinnovi la sottoscrizione del tuo dominio (generalmente una volta all'anno).
```
Comment (OVH-Team-Guides) [inline pages/web/domains/transfer_incoming_generic_domain/guide.it-it.md]: Please check the first words.

File: pages/web/domains/transfer_incoming_generic_domain/guide.pt-pt.md [hunk 7]
```diff
@@ -1,17 +1,17 @@
 ---
 title: 'Transferir o nome de domínio para a OVHcloud'
 slug: transferir-o-dominio-generico
-excerpt: 'Saiba como transferir um nome de domínio para a OVHcloud'
+excerpt: 'Descruba como transferir um nome de domínio para a OVHcloud'
```
Comment (OVH-Team-Guides) [inline pages/web/domains/transfer_incoming_generic_domain/guide.pt-pt.md]:
```suggestion
excerpt: 'Saiba como transferir um nome de domínio para a OVHcloud'
```

File: pages/web/domains/transfer_incoming_generic_domain/guide.pt-pt.md [hunk 8]
```diff
@@ -20,11 +20,13 @@ O seu domínio está atualmente registado num agente de registo e deseja transfe
 
 A transferência de um domínio permite alterar o agente registador que o gere. Pode transferir o seu domínio para a OVHcloud através da criação de uma encomenda. Esta operação demorará entre um a dez dias.
 
-**Saiba como transferir um domínio genérico para a OVHcloud.**
+**Descruba como transferir um domínio genérico para a OVHcloud.**
```
Comment (OVH-Team-Guides) [inline pages/web/domains/transfer_incoming_generic_domain/guide.pt-pt.md]:
```suggestion
**Saiba como transferir um domínio genérico para a OVHcloud.**
```

File: pages/web/domains/transfer_incoming_generic_domain/guide.es-us.md (hunk 5 above)
Comment (Kazarus4) [reply pages/web/domains/transfer_incoming_generic_domain/guide.es-us.md]: done

Comment (Kazarus4) [reply pages/web/domains/transfer_incoming_generic_domain/guide.es-us.md]: corrected

File: pages/web/domains/transfer_incoming_generic_domain/guide.es-es.md (hunk 1 above)
Comment (Kazarus4) [reply pages/web/domains/transfer_incoming_generic_domain/guide.es-es.md]: corrected

File: pages/web/domains/transfer_incoming_generic_domain/guide.it-it.md (hunk 6 above)
Comment (Kazarus4) [reply pages/web/domains/transfer_incoming_generic_domain/guide.it-it.md]: corrected

**Comment by Kazarus4** [comment]
Corrections are done. It still DE and ENGB subs.

**Review by bs-ovh — APPROVED** [review]

**Pull request merged**